sass/MagicBox/MagicBox.scss

```diff
@@ -128,12 +136,11 @@
     top: 100%;
     left: -1px;
     right: -1px;
-    overflow: hidden;
+    overflow: visible;
```

Comment:
Do you have an idea as to why this was hidden previously? Maybe it can break the UI in some cases... But I guess with your refactor it should be fine

Reply:
From all of my testing and what I saw, I didn't see anything out of the ordinary.